index.html

```diff
@@ -395,6 +396,35 @@
       </ul>
     </section>
   </section>
+  <section id="privacy" class="informative">
+    <h2>Privacy and Security</h2>
+    <p>Preload is a declarative fetch primitive that initiates early fetch of
+    resources and separates fetching from resource execution. In effect, it is
+    conceptually similar to initiating a scripted fetch for a resource, but
+    with additional constraints and benefits:</p>
+    <ul>
+      <li>The resource declaration is visible via HTTP headers or HTML markup,
+      which allows the user agent to initiate the fetch earlier, and without
+      blocking on JavaScript execution.</li>
+      <li>The response is not executed and there is no callback to retrieve its
+      payload. The response is retrieved and executed by initiating another
+      fetch (via script, DOM element, etc) that matches the preloaded
+      response.</li>
+      <li>The developer can specify the target context that will consume the
+      response via `as` attribute, which allows the user agent to enforce the
+      relevant CSP policies when initiating the preload fetch. If `as` is
+      omitted, preload defaults to same security and privacy processing as a
+      call to `fetch()` - i.e. subject to `connect-src`.</li>
+    </ul>
+    <p>The site authors are encouraged to take the necessary precautions and
```

Comment:
I see what you're saying, but unless the user bails early from the page, preload is not supposed to download any resources that wouldn't have been downloaded anyway later on.

We could highlight the information leakage that resource download creates here, but as you say, it is true for pretty much any resource fetch.